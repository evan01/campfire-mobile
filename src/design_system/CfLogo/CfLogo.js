// @flow
import React from "react";
import { Text, View, StyleSheet } from "react-native";
import colors from "../styles/colors";
import fonts from "../styles/fonts";

const CfLogo = (props) => (
  <View style={styles.logoContainer}>
    <Text>
      <Text style={styles.camp}>Camp</Text>
      <Text style={styles.fire}>Fire</Text>
    </Text>
  </View>
);

CfLogo.displayName = "TextInput";

const styles = StyleSheet.create({
  logoContainer: {
    height: 50,
  },
  camp: {
    fontFamily: fonts.ROBOTO.LIGHT,
    fontSize: fonts.SIZE.LOGO,
    color: colors.ORANGE.CF_GOLD,
  },
  fire: {
    fontFamily: fonts.ROBOTO.LIGHT,
    fontSize: fonts.SIZE.LOGO,
    color: colors.ORANGE.CF_FIRE,
  },
});
export default CfLogo;
