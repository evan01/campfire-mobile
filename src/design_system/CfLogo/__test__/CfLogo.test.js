// @flow
import React from "react";
import renderer from "react-test-renderer";
import CfLogo from "../CfLogo";

describe("CfLogo", () => {
  test("renders correctly", () => {
    const input = renderer.create(<CfLogo name="bug-report" />);
    expect(input).toMatchSnapshot();
  });
});
