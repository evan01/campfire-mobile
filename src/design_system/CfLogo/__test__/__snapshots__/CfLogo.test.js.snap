// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CfLogo renders correctly 1`] = `
<View
  style={
    Object {
      "height": 50,
    }
  }
>
  <Text>
    <Text
      style={
        Object {
          "color": "#EE964B",
          "fontFamily": "Roboto-Light",
          "fontSize": 38,
        }
      }
    >
      Camp
    </Text>
    <Text
      style={
        Object {
          "color": "#F95738",
          "fontFamily": "Roboto-Light",
          "fontSize": 38,
        }
      }
    >
      Fire
    </Text>
  </Text>
</View>
`;
